tests/appveyor/mariadb.appveyor.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You can remove this file, as we currently do not have MariaDB in the appveyor build matrix